src/libraries/System.Transactions.Local/src/System/Transactions/TransactionManager.cs

```diff
@@ -391,6 +391,8 @@ public static TimeSpan MaximumTimeout
             }
         }
 
+        public static bool ImplicitDistributedTransactions { get; set; }
```

Comment:
Nit: please add XML comments (we use them to seed the docs).

Reply:
And please add them in main too.